root.css

```diff
@@ -0,0 +1,18 @@
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@500;600;700&family=Yellowtail&display=swap');
+:root {
+    --primary-color: #43AA8B  !important;
```

Comment:
do not use !important

Reply:
I have to use !important as i am using different color palette and font style than component library. If i remove !important CL's color pallete will overwrite my website's color pallete.